projects/storefrontlib/src/lib/cms-lib/address-book/address-book.component.html

```diff
@@ -1,67 +1,50 @@
-<div class="cx-page">
-  <section class="cx-page__section container">
-    <div class="row" *ngIf="(addresses$ | async) as addresses">
```

Comment:
Was this `*ngIf` removed on purpose? It probably prevented the DOM sub-tree from being rendered before data are loaded.

Reply:
Ah, there it is. The missing piece I was looking for. I accidentally removed it, and then found an issue that I thought I fixed. I'll have a 2nd look and commit a change.